.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,53 @@
+## Description
+<!-- Briefly describe the bug you are trying to fix -->
+
+<!-- Describe your changes in detail -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Is this PR following the [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format (described below)?
+- [ ] Have you added the unit tests to prevent issue reoccurance( using [pytest fixture pattern](
+      (https://docs.pytest.org/en/latest/fixture.html#fixture))?
+- [ ] Have you run the unit tests using pytest?
+- [ ] Is your code style correct (are you using PEP8, pylint)?
+
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- Read contributing guidelines carefully before submitting the PR. It is available
+  at https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md.
+
+- Read and abide by the Code of Conduct. It is available at  
+  https://github.com/arviz-devs/arviz/blob/master/CODE_OF_CONDUCT.md
+
+- Do not create the PR out of master, but out of a separate branch.
```

Comment:
I would remove this point, it is not really relevant at PR submission, only before.